src/Result/LightningInvoice.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+use BTCPayServer\Util\PreciseNumber;
+
+class LightningInvoice extends AbstractResult
+{
+    public function getId(): ?string
+    {
+        return $this->getData()['id'];
+    }
+
+    public function getStatus(): ?string
```

Comment:
as this is an Enum it is "Unpaid" etc but not NULL afaics on the API docs?

Reply:
same as above

Reply:
Sorry just realizing now that this is invoice not payment, my bad. Will remove.